github/ci/prow-deploy/kustom/base/configs/current/plugins/plugins.yaml

```diff
@@ -317,6 +317,10 @@ plugins:
   - lgtm
   - owners-label
 
+  RHsyseng/rhcos-slb:
+  - trigger
+  - override
```

Comment:
Maybe a few more plugins can be useful here, like `lgtm`, `approve` and maybe `dco`, wdyt?

Reply:
I actually don't think it's really needed. This is not a classic development repo, so I don't think these will be used. 
But of course we can always insert it in a later time if we see we need it. wdyt?